iree/tools/opt_main.cc

```diff
@@ -0,0 +1,149 @@
+//===- mlir-opt.cpp - MLIR Optimizer Driver -------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Main entry function for mlir-opt for when built as standalone binary.
```

Comment:
Please update this comment

Maybe something like:

```suggestion
// Main entry function for iree-opt and derived binaries.
//
// Based on mlir-opt but without registering passes and dialects we don't care about.
```

Reply:
Thanks, done.